pages/index.tsx

```diff
@@ -19,6 +20,9 @@ const Home: NextPage = () => {
           <Grid.Col span={6}>
             <ExportURLInput />
           </Grid.Col>
+          <Grid.Col span={6}>
+            <DataRequirementsViewer />
+          </Grid.Col>
```

Comment:
The reason that we had span `6` in the above two is because webpages are typically divided up into 12 columns when you're talking about the grid system. So two columns that span 6 should take up the whole width. 

If we want the DR viewer to appear on a separate row, we should be explicit about that in the grid system instead. This might also help with that width issue if we look deeper into it..